tensorflow/lite/examples/python/label_image.py

```diff
@@ -57,9 +55,15 @@ def load_labels(filename):
       '--input_std',
       default=127.5, type=float,
       help='input standard deviation')
+  parser.add_argument(
+      '--num_threads',
+      default=1, type=int,
```

Comment:
Would prefer that the default be None, if possible?

Reply:
done